### PR TITLE
Moved CONTACT_* constants to Friendica\Model\Contact class, lesser in global namespace

### PR DESCRIPTION
**Pull request by Quix0r**
This PR moves all `CONTACT_*` constants from `boot.php` to `Friendica\Model\Contact` class. Also a fix for addon repository is finished and **depends** on this change!

### CODE REVIEW

File: src/Model/Contact.php [hunk 1]
```diff
@@ -30,6 +30,19 @@
  */
 class Contact extends BaseObject
 {
+	/**
+	 * @name Contact_is
+	 *
+	 * Relationship types
+	 * @{
+	 */
+	const CONTACT_IS_FOLLOWER = 1;
```
Comment (MrPetovan) [inline src/Model/Contact.php]: Please rename to just `FOLLOWER`.

File: src/Model/Contact.php [hunk 2]
```diff
@@ -30,6 +30,19 @@
...
+	const CONTACT_IS_SHARING  = 2;
```
Comment (MrPetovan) [inline src/Model/Contact.php]: Please rename to just `SHARING`

File: src/Model/Contact.php [hunk 3]
```diff
@@ -30,6 +30,19 @@
...
+	const CONTACT_IS_FRIEND   = 3;
```
Comment (MrPetovan) [inline src/Model/Contact.php]: Please rename to just `FRIEND`.

File: src/Model/Contact.php (hunk 1 above)
Comment (Quix0r) [reply src/Model/Contact.php]: Hmm, I would prefer nice prefixes. Like `User::USER_STATUS_LOCKED` or so.

Comment (Quix0r) [reply src/Model/Contact.php]: Okay, changed my mind.

Comment (MrPetovan) [reply src/Model/Contact.php]: 3 minutes, new world record!

More seriously, you already have the class name prefix, and it's always going to be there, so the double notation isn't necessary.

Comment (Quix0r) [reply src/Model/Contact.php]: And I would also like to do it with the `NETWORK_*` constants. Any suggestions? There is no `Network` class where they could nicely reside.

And same with `PAGE_*`.

Comment (MrPetovan) [reply src/Model/Contact.php]: `NETWORK_*` constants already have been moved to `Friendica\Core\Protocol`, although the legacy constants have been kept for backward compatibility.

`PAGE_*` constants should be moved to `Friendica\Model\Profile`.

Comment (Quix0r) [reply src/Model/Contact.php]: Okay, new PR for this.

Comment (Quix0r) [reply src/Model/Contact.php]: And how about `ACCOUNT_TYPE_*`? Also to same class?

**Comment by MrPetovan** [comment]
Did you run this code on your server? Did you check all modified files for the relevant `use` statement?

**Comment by Quix0r** [comment]
Yes, they are now "live".

**Pull request merged**